bootstrapvz/providers/kvm/__init__.py

```diff
@@ -12,14 +12,20 @@ def resolve_tasks(taskset, manifest):
     taskset.update(task_groups.get_standard_groups(manifest))
 
     taskset.update([tasks.packages.DefaultPackages,
-                    loopback.AddRequiredCommands,
-                    loopback.Create,
                     initd.InstallInitScripts,
                     ssh.AddOpenSSHPackage,
                     ssh.ShredHostkeys,
                     ssh.AddSSHKeyGeneration,
-                    image.MoveImage,
                     ])
+    if manifest.volume.get('logicalvolume', []):
+        taskset.update([logicalvolume.AddRequiredCommands,
```

Comment:
Come to think of it, there might not be any. This is pretty kvm specific, since the volume is kind of tied to the host machine, correct?  
I'm ok with merging this if you squash your commits :-)